src/collections/_documentation/platforms/python/pyspark.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Apache Spark
+sidebar_order: 9
+---
+
+{% version_added INSERT_VERSION_HERE %}
+
+<!-- WIZARD -->
+The Spark Integration adds support for the Python API for Apache Spark, [PySpark](https://spark.apache.org/).
+
+**This integration is experimental and in an alpha state**. The integration API may experience breaking changes in further minor versions.
+
+### Driver
+
+The spark driver integration is supported for Spark 2 and above.
+
+To configure the SDK, initialize it with the integration before you create a `SparkContext` or `SparkSession`.
+
+```python
+import sentry_sdk
+from sentry_sdk.integrations.spark import SparkIntegration
+
+if __name__ == "__main__":
+    sentry_sdk.init("___PUBLIC_DSN___", integrations=[SparkIntegration()])
+
+    spark = SparkSession\
+        .builder\
+        .appName("ExampleApp")\
+        .getOrCreate()
+    ...
+```
+
+### Worker
+
+The spark worker integration is supported for Spark 2.4.0 and above.
+
+Create a file called `sentry-daemon.py` with the following content:
+
+```python
+import sentry_sdk
+from sentry_sdk.integrations.spark import SparkWorkerIntegration
+import pyspark.daemon as original_daemon
+
+if __name__ == '__main__':
+    sentry_sdk.init("___PUBLIC_DSN___", integrations=[SparkWorkerIntegration()])
+    original_daemon.manager()
+```
+
+In your `spark_submit` command, add the following configuration options so the spark clusters can use the sentry integration.
+
+| Command Line Options | Parameter                                | Usage                                                           |
+|----------------------|------------------------------------------|-----------------------------------------------------------------|
+| --py-files           | sentry_daemon.py                         | Sends the `sentry_daemon.py` file to your Spark clusters        |
+| --conf               | spark.python.use.daemon=true             | Configures Spark to use a daemon to execute it's Python workers |
+| --conf               | spark.python.daemon.module=sentry_daemon | Configures Spark to use the sentry custom daemon                |
+
+```bash
+./bin/spark-submit \
+    --py-files sentry_daemon.py \
+    --conf spark.python.use.daemon=true \
+    --conf spark.python.daemon.module=sentry_daemon \
+    example-spark-job.py
+```
+
+<!-- ENDWIZARD -->
+
+## Behaviour
```

Comment:
not sure if we do british vs american spelling, @MimiDumpling ?

Reply:
@untitaker American spelling. 😸 Thanks!